add2cal/add2cal.py

```diff
@@ -65,6 +68,7 @@ def google_calendar_url(self):
             'uid': self.event_uid,
             'sf': 'true',
             'ctz': self.timezone,
+            'add': self.attendee_email
```

Comment:
syntax error. I recommend running the tests in this repo (run pytest in the add2cal directory)